### PR TITLE
Upgrade to rails 5

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -24,18 +24,19 @@ gem 'newrelic_rpm'
 gem 'nokogiri'
 gem 'output-templates', '~> 4.4.3', github: 'guidance-guarantee-programme/output-templates'
 gem 'phoner'
-gem 'princely'
 gem 'postcodes_io'
+# Switch back to the original after this is merged:
+# https://github.com/mbleigh/princely/pull/52
+gem 'princely', github: 'AndrewVos/princely', branch: 'remove-alias-method-chain'
```
Comment (Guntrisoft) [inline Gemfile]: Can you not fork it on the pension guidance space?

File: config/routes.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
-Rails.application.routes.draw do
+Rails.application.routes.draw do # rubocop:disable Metrics/BlockLength
```
Comment (dwhenry) [inline config/routes.rb]: Seems this we could just have one of these are the top of the file.

Comment (benlovell) [inline config/routes.rb]: It has exactly the same effect either way.

File: Gemfile (hunk 1 above)
Comment (benlovell) [inline Gemfile]: As a general rule - if you do the work on pension wise's clock you should maintain those changes on our fork, not your personal one.

File: Gemfile [hunk 3]
```diff
@@ -24,18 +24,19 @@ gem 'newrelic_rpm'
...
 gem 'puma'
-gem 'rack-contrib'
-gem 'rails', '4.2.7.1'
+gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
```
Comment (benlovell) [inline Gemfile]: This is weird. Can we just pin to `5.0.0.1` instead.

File: app/controllers/booking_requests_controller.rb [hunk 4]
```diff
@@ -27,8 +27,7 @@ def complete
     end
   end
 
-  def completed
-  end
+  def completed; end
```
Comment (benlovell) [inline app/controllers/booking_requests_controller.rb]: Ew. I turned this cop off on the other projects. Can we do that here too?

File: Gemfile (hunk 1 above)
Comment (AndrewVos) [inline Gemfile]: :heavy_check_mark:

File: config/routes.rb (hunk 2 above)
Comment (AndrewVos) [inline config/routes.rb]: :heavy_check_mark:

File: app/controllers/booking_requests_controller.rb (hunk 4 above)
Comment (AndrewVos) [inline app/controllers/booking_requests_controller.rb]: Totalamente :heavy_check_mark:

File: Gemfile [hunk 5]
```diff
@@ -24,18 +24,18 @@ gem 'newrelic_rpm'
...
 gem 'output-templates', '~> 4.4.3', github: 'guidance-guarantee-programme/output-templates'
 gem 'phoner'
-gem 'princely'
 gem 'postcodes_io'
+# https://github.com/mbleigh/princely/pull/53
+gem 'princely', github: 'guidance-guarantee-programme/princely', branch: 'remove-alias-method-chain'
 gem 'puma'
-gem 'rack-contrib'
-gem 'rails', '4.2.7.1'
+gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
```
Comment (benlovell) [inline Gemfile]: Can we just pin this to `5.0.0.1`?

**Comment by benlovell** [comment]
Any chance we could rebase this and update to Rails 5.0.1? Keen to try and see this one through.

**Comment by AndrewVos** [comment]
@benlovell It's on `5.0.1` now ^^

**Pull request merged**